### DISCUSSION TITLE
第3节课笔记(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#265] [笔记]
请在下方提交第 3 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by jiangye520** [comment]
https://mp.weixin.qq.com/s/pbNiOVJjptuI7wKdk7qXAA

**Comment by Tian-CASIA** [comment]
https://blog.csdn.net/qq_55090315/article/details/135491848?spm=1001.2014.3001.5501

**Comment by admlrz** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135515331

**Comment by dearSocrates** [comment]
https://blog.csdn.net/rabbit9798/article/details/135464373?spm=1001.2014.3001.5501

**Comment by lishuang70345** [comment]
https://blog.csdn.net/weixin_42748604/article/details/135514475

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec3

**Comment by adela778** [comment]
https://blog.csdn.net/Adela0546/article/details/135575846

**Comment by zhangyutuan** [comment]
https://www.processon.com/view/link/65abe99b7fce4a25f8d9f65d
